src/component/Button.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class Button extends Component {
+    render() {
+        const { onClick, children } = this.props;
+
+        return <button className="button" onClick={onClick}>{children}</button>;
+    }
+}
```

Comment:
Voor de volgende keer **overal** een functional component gebruiken in plaats van een class component 😉